ui/apps/platform/src/Containers/ComplianceEnhanced/ClusterCompliance/ScanConfigs/components/ScanConfigParameters.tsx

```diff
@@ -0,0 +1,67 @@
+/* eslint-disable no-nested-ternary */
```

Comment:
```suggestion
/* eslint-disable no-nested-ternary */
```
This looks unused too.

(Honestly there are enough of these comments that I'd say we can disable the rule entirely.)